userspace/libsinsp/sinsp.h

```diff
@@ -927,18 +925,18 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 	}
 #endif
 
-	static inline bool simple_consumer_consider_evtnum(uint16_t etype)
+	static inline bool should_consider_evtnum(uint16_t etype)
 	{
 		enum ppm_event_flags flags = g_infotables.m_event_info[etype].flags;
 
-		return ! (flags & sinsp::simple_consumer_skip_flags());
+		return ! (flags & sinsp::event_skip_flags());
 	}
 
-	static inline bool simple_consumer_consider_syscallid(uint16_t scid)
+	static inline bool should_consider_syscallid(uint16_t scid)
 	{
 		enum ppm_event_flags flags = g_infotables.m_syscall_info_table[scid].flags;
 
-		return ! (flags & sinsp::simple_consumer_skip_flags());
+		return ! (flags & sinsp::event_skip_flags());
 	}
```

Comment:
I think we can remove that since the `syscall_table` has no `EF_SKIPPARSERESET | EF_UNUSED | EF_OLD_VERSION` flags, they are related to the event not to the syscall